doc/api/pymongo/collection.rst

```diff
@@ -58,9 +58,134 @@
       .. automethod:: create_index
       .. automethod:: create_indexes
       .. automethod:: drop_index
+      .. describe:: reindex
+
+         Removed. Run the `reIndex command`_ directly instead. Code like this::
+
+           >>> result = database.my_collection.reindex()
+
+         can be changed to this::
+
+           >>> result = database.command('reIndex', 'my_collection')
+
+         .. _reIndex command: https://docs.mongodb.com/manual/reference/command/reIndex/
       .. automethod:: drop_indexes
       .. automethod:: list_indexes
       .. automethod:: index_information
       .. automethod:: drop
       .. automethod:: rename
       .. automethod:: options
+      .. describe:: map_reduce || inline_map_reduce
+
+         Removed. Migrate to :meth:`~pymongo.collection.Collection.aggregate` or run the
+         `mapReduce command`_ command directly with :meth:`~pymongo.database.Database.command`
+         instead. For more guidance on this migration see:
+
+         - https://docs.mongodb.com/manual/reference/map-reduce-to-aggregation-pipeline/
+         - https://docs.mongodb.com/manual/reference/aggregation-commands-comparison/
+
+         .. _mapReduce command: https://docs.mongodb.com/manual/reference/command/mapReduce/
+      .. describe:: parallel_scan
+
+         Removed. MongoDB 4.2 removed the `parallelCollectionScan command`_.
+         There is no replacement.
+
+         .. _parallelCollectionScan command: https://docs.mongodb.com/manual/reference/command/parallelCollectionScan/
+      .. describe:: initialize_ordered_bulk_op || initialize_unordered_bulk_op
+
+         Removed. Use :meth:`pymongo.collection.Collection.bulk_write`
+         instead. Code like this::
+           batch = coll.initialize_(un)ordered_bulk_op()
```

Comment:
Not sure if it would be better to separate into two sections. Saying "(un)ordered" makes sense to me but might not be clear to the user.